content/blog/convince-my-boss.md

```diff
@@ -0,0 +1,31 @@
+
+## Important notes:
+1. You do not need to keep all the points. Remove ones you deem unnecessay. The shorter it is, the better the chances that your boss will read it.
+
+2. Watchout for the {TEMPLATE_VARIABLES}. You need to replace it with information relevant to you. Don't let one escape you.
+
+## The letter
+
+Dear {BOSSNAME},
+
+I am excited to brief you about PyCon India, India's premier conference on using and developing the Python programming language that takes place in Chennai from October 12-15, 2019. About a 1000+ of the industry's Pythonistas from different fields are bound to be in attendance for the 11th edition of this mega event.
+
+It would be great if I get an opportunity to attend this conference as there will be innumerable benefits for me as an individual which will in turn impact the organization as well. To list a few:
+
+* (Optional) Am Particularly keen on attending this {TALK/WORKSHOP/DEVSPRINT} on {TOPIC} presented by {SPEAKER-NAME} as our project is based on this {OR UPCOMING PROJECT BASED ON THIS}.
+* Exposure to the actionable best practices & tools that we could use in our projects, saving money and time spent exploring different options.
+* Provides a forum to get a sneak peek at upcoming innovations.
+* Provides a platform to unmask the rationality behind the trending technologies like ML/AI, Automation, Web development, Software development, IOT and more that uses Python.
+* Learning Technologies from a wide array of Talks, Workshops, Keynotes speech, Developer Sprints each presented by inspiring and enthusiastic Industry experts, community leaders and many young minds delving into new technologies.
+* Opportunity to network with like-minded people from diverse backgrounds that includes Geek Programmers, Enthusiasts, Entrepreneurs, Scholars, Intellectual and many more.
+* A chance to spread the word about our Company and create goodwill about the opportunities and incentives the employees are given. This will be the ideal way to attract the future employees from an diversified talent pool.
+* {OPTIONAL - } As a _______{WOMAN/, attending the conference will also demonstrate to the tech community that our company takes diversity in our workforce very seriously. Innovative thinking and better products come from teams that are diverse covering more ground.
+
+There is an Early Bird tickets offer on for the first 300 tickets sold before 30th June 2019 and it costs only Rs 1100. Booking later (or after 300 tickets)would cost Rs 1600. The tickets include access to all PyCon India keynotes & talks, sponsor showcase, community spaces {if included this time}, an official PyCon India T-shirt along with lunch and high-tea in the evening for both days (12th-13th October).
+
```

Comment:
Need to add a space after the close brace.
community spaces {if included this time} -> open spaces